cms/src/components/content/ContentOverview.vue

```diff
@@ -41,7 +41,7 @@ type Props = {
 const props = defineProps<Props>();
 
 const languages = db.whereTypeAsRef<LanguageDto[]>(DocType.Language, []);
-const selectedLanguage = ref<Uuid>("");
+const selectedLanguage = ref<Uuid>(localStorage.getItem("cms_selectedLanguage") || "");
```

Comment:
Move the selectedLanguage ref to globalConfig. Might have to convert globalConfig to a module (and not a Pinja store)